src/devices/CharacterLcd/ICharacterLcd.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Iot.Device.CharacterLcd
+{
+    /// <summary>
+    /// Interface for character LCD Low-Level handler
+    /// </summary>
+    public interface ICharacterLcd : IDisposable
+    {
+        /// <summary>
+        /// Enable/disable the backlight. (Will always return false if no backlight pin was provided.)
+        /// </summary>
+        bool BacklightOn { get; set; }
+
+        /// <summary>
+        /// Enable/disable the display.
+        /// </summary>
+        bool DisplayOn { get; set; }
+
+        /// <summary>
+        /// Enable/disable the underline cursor.
+        /// </summary>
+        bool UnderlineCursorVisible { get; set; }
```

Comment:
What's this? Should this be joined with BlinkingCursorVisible (i.e. CursorType enum)

Reply:
This could be done, of course. There exist two types of cursors: One underline cursor (using the last row of a character cell) and one block cursor, that fills the whole character cell. I can create an enum with the possible modes.

Reply:
@krwq If I change this, it would likely imply a breaking change on Hd44780, because I don't think we should use an enum in the interface but keep the bool's in the implementation. Or what do you think? 
Same applies for comment https://github.com/dotnet/iot/pull/859#discussion_r381588322